builder-function.Dockerfile.j2

```diff
@@ -0,0 +1,20 @@
+FROM {{docker_account}}/installer-base:{{installer_base_tag}}
+
+# install general packages
+ENV S11 https://raw.githubusercontent.com/binaris/711/master
+RUN apt-get update && apt-get install -y curl git vim
+RUN curl -sL $S11/ubuntu/node/10.x.sh | bash
```

Comment:
why not 12?

Reply:
@michaeladda I assume `node 12`, please verify that this is a mistake so I can update or take the appropriate measures to make sure `node 10` is not a problem